IRremoteESP8266.cpp

```diff
@@ -201,7 +201,6 @@ void IRsend::sendLG (unsigned long data, int nbits) {
   // Header
   mark(LG_HDR_MARK);
   space(LG_HDR_SPACE);
-  mark(LG_BIT_MARK);
```

Comment:
we should have mark,space,mark I think. was always there. its still there in the Arduino lib